Lab 7.ipynb

```diff
@@ -0,0 +1,854 @@
OVt5Z8rCY6vVxdKZHalk/oEmzquuIhBMIzMftlW9xmvn/Qmdgcm4RBuzs3ZzqXuFyzpewULZlM+t8IuffuCE79XvbX6KXx9pJhbwVwEkwJzYdubWb2dhVYTiSWcl7f/rr9/PkfqncbEuLOYIEORbeX/NwWB5T0AIegW4/AFYRewQV8S9+/LR//EtAFoP1fHbIw0c3dNJV6id9Ez/uJK5BAvfBcBTs6Bvcr6nvL2OkYXrRSAa30+k5VUircOfBnWqGKx20CHLhTRoCww+7oikdRDOyKR4RvOg+47ElPzfEw4fIRJJfSpYlBjhcJi00sEfxL2xfT77OyYxq6Bh0H2TOdGYJ5kfdV7MlOYL6LSBM5oWWg5zLIf0SQkCBceyXRLmLbbeLeE6ca7vtLPu59HdH+x7Wze1nM2lfcd+5mfMuF65p6Fwe8/KpN3Th47p/2H1+Ho7N8XfIG1XhLT9ycd6AEsc3Bh8garCS5k05Zq+wax+Em4qjksYqPZL54zVRMrXk8t2zmU79Pud/mzGRbyScR4JJgHn9bR3ta+lo3Ez8Uj3ed8E3uSaDU1ctq2FNKKctynec0R3IAWgNbed9uwOEq6ThJ/hW1Kxh6KpBwgQ5IifPfVyejarw31fW2tVESteNi7fsx6AYtdApCCdzV1eUfCYQXu877S3WTnz2TT5Q/y0+B46J8VorOv0glhdcGfiGW5KvAbF8NKsY1PNPrQ1yAp/NbedWTPYnz6ZgqmbySvZAezm5+kr/ClixX7dpAV8MFbKsqq3Odoa5n2Z5zE/uISD5zVzMNxKQflWikt+1tOn12qW8kR9AeCVaWjPPsBFW3ax4rftnJPXTFfuUp68bC47582isz5AU30XLhDrCbh186ambWLLnrWkrUvQnLeCrsxi5mTmUHrhTs7O/zFla6dTMj2bTefN4IX0L2BLLOm4ALzsoZzyOC+VrwBW9MkgguSFro9nxF0WF2S1cef0+YMGiSZCZryLIAlcIkA0HD9uuxHgouJzaF1V3SeAdTozl2x+xklk2bJlbs2aNYPvOESPfPCvaMlaSuGSk7do8cnm0NYfE2k/TDhreIO/qJ9WXTRt9Fdt2+/CdGI9S3sOR08gKcmAbkPZTLZNGd7Sqh9seYz31K3CpVBAMri1k+Bub467i7u+9Z176Z2tcCAcp9McGUnSiXv/Eu6t/xOxjumv0lU+8P9n3oDnxE+mBmL+cRkHl5C5f+AP9/39bslm3t3yGjku+ZPacMcyaFtBS8LLjkpPWE8N7CemZfLbcu8f+fy4l5Y+s8pRcmgPCdcJLp20zEMQiBDobGdvwX7ezm9mWscUJkXbqMo7TLsLM/1IiPQjiwiE0lh8dBdLbS9TYm1k1ENo8hU8vuADrJqSw84y7+82x8XJwDj7cA0zj2xlUtlm8kt2kSBB3MVxiRixRIDN1dOJrTqLi6vWU2SNbJ6xmMT8Kix4Loc7ikg4R04iwNbyaWzya06Ewm09A81u3dPPAOZGd2AYQbx08LgfWAqFvPdT76c8wVAX+5lFRewQf9P6HYLOKK1cRnHHrRS/e/6QfwH+zA9O9Pb+kkl8ZGrRAEecWhoefoSNT++gKtR3+ejQ5MksuvFsFl9WPsCRYyfZPYeh3/f//o8/59UZhT3ZVidS3lHPL2755JD6mSozW+ucS/0fCBlzYzX++t6HP0E03kD5opHV0TtZpJfuIaP4QNJtnf5vwIHyjYJ+kKl7eshgAv4y5f0le/DUX6Pl0UwK4y/zpielMmbpdkf8WW5OvIYNVBSnl3gsQTAUILOg75Qx29yO7Tg++yfuIGiQ7S8Dv40I7Tg60/NoDGcPer3m4gVE8vMYaAyzedpZ7Cmfg2E9GaG99Q7y9HZ8wAcCmasJZB6bOriwbgnz6o+9z3s/EALIbXXk+Kfuzjrpys8lkj/w68oMZlEybS/5JbsG3Ae8aVUn+ne9ueptotHkD1/vWNVIcUPyaW0VBe+iIm/xce39A2kbEpPYmRY67t7dmFjN5YlNxAhS234heyJ+RpMv2LqPcEuVn8FEnzo/va/jTVXLP26KWlX7Cmo7zqMp6Gjr2NsvI+qYrvxcNuSfxW6bQji9k7RwJ2DU5+wlktU3ULm8djazWmaQSOugtHQjU0r2ECDY80Ctu8h0dx+ezTFez0l4U9S2J3rG4hlnPU/O7B28mfMeVmVeRNOhADdUvcRN8Td4KSdGIHERVx65gGqL0TRrFXklO3g5fQXP117EzF0ZXNbSSjDYhOtI9LxvaH2Z0rxyyv0C2C0z3+qpaTQS3ePHwTKEUhIwgsEg75+SwWcWT9wKHitXrgTgnHOfAfoW2w53r34Y/gYA9+/6U9L3vkiIGJdXfBzm/gXfsBDXvfxp7p9bzicznuTxc99N8d5v8A9bv8xnb/D+/fjwK5WUZE7j/X9/atWWHO4YbNwDSWZ2HfBvQBD4T+fcCasGjtVA5vt/9g90JWoIZ43u6hAnm47sdLqyBp5SlW4ZpAdSnXLlDYcCweHds3VlZWwqmTLoQCMYbiWYNrQpUc45zIw0G0GSXcJBwAiEj396uDE+HYAlwQO0N6+moy31GgNntewjMx6hMzT4/P6P/7KL0hrH4Sl27AFPr9vVkJ5Lc7jvQLB7kBqdVErbpL5zNd+ZsZAdZTPpPtn765/lusZXe/4Ogtu6CO2OYJbgYEYuhzKyaS08dh7Xa1jlLISzIJnBLNKDmRRM3cImy+GVLujM2sn8unO5qeEiZkxfz7Ohdt44Oi+l+5MgwYLOnZy162if9q78HGL5kwlY8voE02bsJr9kl98/7795qzspXtsK8QAJAgRI0H5enK7zczhcWUzDFsc7wQVsKZlLKBsCFuSSnRu5svAVguenEd4QJn1jBw2BPKpyL6E9cx6tOS3sLt1GVjSTQ+3lTD+4hXzXxNIlATLmNfDVgr9gf3Aq0/wnWd1ZIXOjO3qCOPFYes80qVDIe2Jo0QxWdL3NNZ2rAO8J3ksZS/tkFfXOMOme
```

Comment:
The graphs are pretty cool but they're a bit incomprehensible, either explain or simplify